src/Core/Utilities/EncryptedStringAttribute.cs

```diff
@@ -11,6 +12,9 @@ namespace Bit.Core.Utilities;
 /// </summary>
 public class EncryptedStringAttribute : ValidationAttribute
 {
+    private static readonly SearchValues<char> _pipeChar = SearchValues.Create("|");
+    private static readonly SearchValues<char> _periodChar = SearchValues.Create(".");
```

Comment:
ℹ️ For single values I agree that this doesn't seem to provide too much.

Reply:
Yeah it definitely didn't give the improvement I was hoping for. It's much more optimized for more values, want me to remove this part?

Reply:
Might be a good idea, as we want to show a best practices pattern and not potentially mislead a copy-paster.

Reply:
Removed it.